src/transform_shapes/transform_shapes.py

```diff
@@ -0,0 +1,64 @@
+from abc import ABC, abstractmethod
+
+import numpy as np
+import cv2
+
+from src.shapes.point import Point
+
+
+class TransformShape(ABC):
```

Comment:
this is a good example of possible static class - Transformation... just move the last function to someplace else

Reply:
I can do it static however the mission is to do it OOP and in OOP we want to do it as base object (we can make it util functions however it will make it less OOP)